Sources/Flare/Classes/Models/Internal/SK1StoreProduct.swift

```diff
@@ -42,6 +42,10 @@ extension SK1StoreProduct: ISKProduct {
     var currencySymbol: String? {
         numberFormatter.currencySymbol
     }
+    
+    var locale: Locale {
+        self.product.priceLocale
```

Comment:
`self` is not needed here. Could you please remove the `self` keyword in places where it isn't explicitly needed?

Reply:
Sure